en/components/grid_toolbar.md

```diff
@@ -0,0 +1,173 @@
+</div>
+
+The IgxGrid's toolbar is shown using the grid's `showToolbar` property - just set it to `true`. The toolbar supports a textual title which is left aligned and its content is set using the grid's `toolbarTitle` property. The following code snippet demonstrates how to enable a toolbar and set its title:
+
+```html
+<igx-grid [data]="data" [autoGenerate]="true" height="400px" width="600px"
+    [showToolbar]="true"
+    toolbarTitle="Grid Title">
+</igx-grid>
+```
+
+#### Exporting
+
+The IgxGrid's toolbar can be configured to allow IgxGrid's data exporting in different formats - MS Excel and CSV. You can enable each exporting format independently by setting its dedicated boolean property to true:
+- for MS Excel export set the grid's `exportExcel` property to `true`
+- for CSV export set the grid's `exportCsv` property to `true`
+
+The available export options are hosted in a drop down which is shown when the main export button on the toolbar is pressed. There are properties for configuring each button's text and they are listed in the API section below.
+
+<div class="sample-container loading" style="height:420px">
+    <iframe id="grid-toolbar-sample-2-iframe" src='{environment:demosBaseUrl}/grid-toolbar-sample-2' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="grid-toolbar-sample-2-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+The following code snippet demonstrates how to enable the MS Excel export and CSV export features on the toolbar and also set titles for the exporting buttons:
+
+```html
+<igx-grid [data]="data" [autoGenerate]="true" height="400px" width="600px"
+    [showToolbar]="true"
+    toolbarTitle="Grid Title"
+    [exportExcel]="true"
+    [exportCsv]="true"
+    exportText="Export"
+    exportExcelText="Export to Excel"
+    exportCsvText="Export to CSV">
+</igx-grid>
+```
+
+The export to MS Excel and the export to CSV features are using respectively the `IgxExcelExporterService` and `IgxCsvExporterService` as providers. If you need to use any of them you should specify them in the 'providers' array of your 'app.module.ts' file. For example the following code snippet demonstrates how to include all exporter services:
+
+```typescript
+// app.module.ts
+
+...
+import { IgxExcelExporterService, IgxCsvExporterService } from "igniteui-angular";
+
+@NgModule({
+  providers: [ IgxExcelExporterService, IgxCsvExporterService ]
+})
+
+export class AppModule {}
+```
+
+#### Customizing the Export
+
+When the user initiates an export process the IgxGrid's `onToolbarExporting` event is emitted. This event exposes the export options and the exporter and this allows you to configure the export process.
```

Comment:
"exposes the export options and the exporter and this allows you..." doesn't make sense to me.  Are both the export options and the exporter exposed by this event?  If so, I'd recommend, "This event exposes both the export options and the exporter itself.  This allows you to configure the export process."